src/plans/details/tabs/Details/components/SettingsSection/components/PlanTargetNamespace/TargetNamespaceSelect.tsx

```diff
@@ -0,0 +1,42 @@
+import type { FC } from 'react';
+import useProviderInventory from 'src/modules/Providers/hooks/useProviderInventory';
+
+import { FilterableSelect } from '@components/FilterableSelect/FilterableSelect';
+import type { OpenShiftNamespace, V1beta1Provider } from '@kubev2v/types';
+import { useForkliftTranslation } from '@utils/i18n';
+
+type Props = {
+  provider: V1beta1Provider;
+  value: string;
+  onChange: (val: string) => void;
+};
+
+const TargetNamespaceSelect: FC<Props> = ({ onChange, provider, value }) => {
+  const { t } = useForkliftTranslation();
+
+  const { inventory: namespaces } = useProviderInventory<OpenShiftNamespace[]>({
+    provider,
+    subPath: 'namespaces?detail=4',
```

Comment:
what is the details=4 ?

Reply:
did a little investigation, the inventory API URL needs the ?detail=<number> pattern to get access to the actual object it tries to access, I did see detail=1 and detail=4 and detail=3000 work the same. it's an existing pattern in across the app the do like detail=4, maybe @sgratch knows more on this